### PR TITLE
Hotfix file #58 add SVG support

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix file #58 add SVG support

### CODE REVIEW

File: src/MetaModels/Helper/ToolboxFile.php [hunk 1]
```diff
@@ -933,6 +933,15 @@ private function processFile($fileName)
             }
         }
 
+        // Prepare SVG images.
+        if ($information['isSvgImage'] = $file->isSvgImage) {
+            $information['src'] = $fileName;
+            if (file_exists(TL_ROOT . '/' . $information['src'])) {
+                $size              = getimagesize(TL_ROOT . '/' . $information['src']);
```
Comment (discordier) [inline src/MetaModels/Helper/ToolboxFile.php]: Why are you fetching the size here and discard the value?
Does this really work with SVG?
If so, we should use the size value returned herin or (if specified) override it with the resize information.

Comment (zonky2) [reply src/MetaModels/Helper/ToolboxFile.php]: that´s right - I confused `size` with `sizetext` at template... I´ll kill this rows

**Comment by zonky2** [comment]
@discordier it´s now o.k.?

**Comment by stefanheimes** [comment]
Merged with 26de1410bbc940339a6ec8b805264bb41fcbe866

**Pull request closed without merging**